### PR TITLE
OBSDA-553: add provider name to cluster_infrastructure_provider when external platform

### PR DESCRIPTION
**Pull request by rccrdpccl**
This will help track installations for third party platform (such as OCI) integrating through external platform

### CODE REVIEW

**Comment by rccrdpccl** [comment]
Couldn't really avoid adding another label as it would break compatibility

File: pkg/operator/configmetrics/configmetrics.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ func Register(configInformer configinformers.SharedInformerFactory) {
 		cloudProvider: prometheus.NewGaugeVec(prometheus.GaugeOpts{
 			Name: "cluster_infrastructure_provider",
 			Help: "Reports whether the cluster is configured with an infrastructure provider. type is unset if no cloud provider is recognized or set to the constant used by the Infrastructure config. region is set when the cluster clearly identifies a region within the provider. The value is 1 if a cloud provider is set or 0 if it is unset.",
```
Comment (vrutkovs) [inline pkg/operator/configmetrics/configmetrics.go:21]: Needs `external` explanation

**Review by vrutkovs — APPROVED** [review]
/approve
/lgtm

**Pull request merged**